bpfd-operator/pkg/bpfd-agent/bpfd-agent.go

```diff
@@ -339,6 +352,19 @@ func (r *BpfProgramReconciler) reconcileBpfProgramConfig(ctx context.Context,
 				maps = map[string]string{}
 			}
 
+			// TODO(astoycos) This will need to end up being a list of loadRequests
```

Comment:
So here I don't think we need to build the BPFDLoadRequest just set 

```
			bpfProgram.Spec.Programs[v.Uuid] = bpfdiov1alpha1.BpfProgramMeta{
				AttachPoint: BpfProgramConfig.Spec.AttachPoint,
				Maps:        maps,
			}
```

Then it's easier to move the building of the load request into `r.loadBpfdProgram`